contrib/log_correlation/log4j/src/test/java/io/opencensus/contrib/logcorrelation/log4j/OpenCensusTraceContextDataInjectorTest.java

```diff
@@ -82,4 +89,64 @@ public void useDefaultValueForInvalidSpanSelection() {
       System.clearProperty(OpenCensusTraceContextDataInjector.SPAN_SELECTION_PROPERTY_NAME);
     }
   }
+
+  @Test
+  public void insertConfigurationProperties() {
+    assertThat(
+            toMap(
+                new OpenCensusTraceContextDataInjector()
+                    .injectContextData(
+                        Lists.newArrayList(
+                            Property.createProperty("property1", "value1"),
+                            Property.createProperty("property2", "value2")),
+                        new SortedArrayStringMap())))
+        .containsExactly(
+            "property1",
+            "value1",
+            "property2",
+            "value2",
+            "openCensusTraceId",
+            "00000000000000000000000000000000",
+            "openCensusSpanId",
+            "0000000000000000",
+            "openCensusTraceSampled",
+            "false");
+  }
+
+  @Test
+  public void handleEmptyConfigurationProperties() {
+    assertContainsOnlyDefaultTracingEntries(
+        new OpenCensusTraceContextDataInjector()
+            .injectContextData(null, new SortedArrayStringMap()));
+  }
+
+  @Test
+  public void handleNullConfigurationProperties() {
+    assertContainsOnlyDefaultTracingEntries(
+        new OpenCensusTraceContextDataInjector()
+            .injectContextData(null, new SortedArrayStringMap()));
+  }
+
+  private static void assertContainsOnlyDefaultTracingEntries(StringMap stringMap) {
+    assertThat(toMap(stringMap))
+        .containsExactly(
+            "openCensusTraceId",
+            "00000000000000000000000000000000",
+            "openCensusSpanId",
+            "0000000000000000",
+            "openCensusTraceSampled",
+            "false");
+  }
+
+  private static Map<String, String> toMap(StringMap stringMap) {
+    final ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
+    stringMap.forEach(
```

Comment:
I don't remember project settings, but if this project allows test source to be 1.8+ method reference builder::put is much nicer. In zipkin we do this by using retrolambda on main but keep source level as 1.8. That's only because intellij at least at some point didn't pay attention to source level for test tree.